packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordModal.tsx

```diff
@@ -148,10 +148,14 @@ const DxtradePasswordModal = () => {
 
     if (isMobile) {
         return (
-            <ModalStepWrapper renderFooter={() => renderFooter} title={' '}>
-                {successComponent}
-                {passwordComponent}
-            </ModalStepWrapper>
+            <Modal>
+                <Modal.Header title=' ' />
```

Comment:
Rather than leaving it as an empty string, I suggest title should be optional. Or you can just remove the `Modal.Header `altogether.